src/services/nexboard/services/Boards.js

```diff
@@ -0,0 +1,26 @@
+const nexboardClient = require('../utils/Nexboard')(process.env.NEXBOARD_MOCK_URL);
```

Comment:
Are you sure you want to use the `NEXBOARD_MOCK_URL` here?

Reply:
To override it for test. Maybe it can solved more clarify that not test env added at this position. I will think about it.

Reply:
done